src/main/scala/cromwell/engine/backend/jes/authentication/JesAuthentication.scala

```diff
@@ -20,7 +34,12 @@ object ProductionJesConnection {
  */
 trait JesAuthentication { self: JesConnection =>
 
-  def authenticated[A](f: JesInterface => A) = f(jesConnection)
+  def authenticateAsCromwell[A](f: JesInterface => A) = f(jesCromwellConnection)
+
+  /**
+   * Important note: Will default back to cromwell authentication if the configuration for user authentication has not been set or if the refreshToken has been supplied.
```

Comment:
No because `authenticateAsUser` only gives you back a `GoogleCloudStorage` object, so you can only read/write stuff into GCS with it, not request compute.
So this only allows you to read/write stuff as a user if they give a refresh token, or as cromwell if they don't.